src/lib/snack-bar/simple-snack-bar.scss

```diff
@@ -16,6 +16,14 @@ $mat-snack-bar-button-margin: 48px !default;
   background: none;
```

Comment:
It doesn't let me set a comment on the proper line, but you should be able to get rid of the `@include mat-button-reset` a little bit above.